### ISSUE TITLE
`--textfield` option doesn't respect font size set with `--messagefont`

### ISSUE THREAD

**Issue by wegotoeleven** [bartreardon/swiftDialog#247]
When setting `--messagefont`, any strings set with `--textfield` do not display with the same font size. For example, using the following code presents the following dialog:

```
/usr/local/bin/dialog \
    --title 'FileVault Prompt' \
    --message 'To set up FileVault, please enter the password for "John Appleseed".\n\n' \
    --small \
    --titlefont 'size=13' \
    --messagefont 'size=13' \
    --messageposition top \
    --icon /System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/FileVaultIcon.icns \
    --textfield 'Password,prompt=Enter your computer password here,required'
```
![SCR-20230327-njyj](https://user-images.githubusercontent.com/7345765/227963699-833f639f-c761-41b7-a9cf-0f82d86136e4.png)

I would be expected that the string set in `--textfield` should respect the message font size configured in `--messagefont`

**Comment by bartreardon** [comment]
2.2 will manage textfield in relation to `--messagefont size`. 

By default labels are 16pt and message is 20pt. Labels will inherit 4 points below whatever message size is unless it's < 20 point in which case both will be the same.

**Comment by bartreardon** [comment]
forgot to mark this as closed after there 2.2 release.
